app/pharmacy/views/dashboard.html

```diff
@@ -2,6 +2,8 @@
 
 <visit-header patient="vm.patient"></visit-header>
 
+<check-in patient="vm.patient" on-check-in="vm.reload();"></check-in>
```

Comment:
`on-check-in` binding was changed to `on-check-in-change` in https://github.com/esaude/esaude-emr-poc/pull/540.